### ISSUE TITLE
Where is angular-animate used?

### ISSUE THREAD

**Issue by distributev** [distributev/the-app#94]
![008 - angular-animate](https://cloud.githubusercontent.com/assets/19224635/17680687/c41a850a-6341-11e6-812f-ecf44dc02a47.png)

**Comment by luismanuel001** [comment]
This just came from merging the ng-mail-merge-directive dependencies, not sure where it is used

**Comment by distributev** [comment]
Yes I know but I need to understand. The mail-merge GUI was tested well initially - now I will test even more in depth since now we build the complete end2end solution.
